src/main/java/io/a2a/spec/CancelTaskRequest.java

```diff
@@ -37,4 +37,35 @@ public CancelTaskRequest(@JsonProperty("jsonrpc") String jsonrpc, @JsonProperty(
         this.method = method;
         this.params = paramsMap;
     }
+
+    public static class Builder {
+        private String jsonrpc;
```

Comment:
Not sure if this is handled somewhere, but it says that the default for this is "2.0". Mentioning this since I am trying to deal with the same on the server side

Reply:
Good point, will address that separately.